web-wallet/src/routes/(app)/dashboard/staking/+page.svelte

```diff
@@ -32,7 +21,7 @@
     reverse
   >
     <StakeContract
-      descriptor={contractDescriptors[1]}
+      descriptor={contractDescriptors[2]}
       on:operationChange={({ detail }) => updateOperation(detail)}
       on:suppressStakingNotice={() => updateSetting("hideStakingNotice", true)}
```

Comment:
You removed the function that was called here.

Reply:
I don't follow.  I just updated the contractDescriptors.  Show me tomorrow what you are talking about here.